config/routes.rb

```diff
@@ -3,6 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :merchants, only: [] do
```

Comment:
Can you explain what the only: [] and module: "merchant" pieces of code are doing here?